### ISSUE TITLE
Support the development and continued operation of widefield facilities as transient discovery machines

### ISSUE THREAD

**Issue by ddobie** [ethrane/transients#21]
This is an attempt at a broad point (perhaps _too_ broad) that synthesises some of the discussion at the Sydney town hall and some of the issues already raised here. The general consensus seems to be that wide field of view is the next step in the time domain arsenal, Australia is already leading that effort in a range of areas and is well-placed to continue leading that over the next decade.

* Rubin is the next big thing in optical time domain astronomy, and Australia is already heavily involved (#13).
* Widefield instruments are necessary for GW follow-up, which overlaps with some of the proposed key questions
* Most of the arguments for radio facilities in #17 (which inspired the phrasing of this point!) apply to widefield facilities across the entire spectrum more broadly
* We're already leading the current generation of widefield radio facilities (ASKAP/MWA) and have the opportunity to shape the next generation of them (e.g. #20)
* As Ron argued, widefield facilities are the only way to discover the unknown - FRBs might never have been discovered without them

**Comment by RonEkers** [comment]
I agree that all widefield instruments play a key role in opening new discovery space.  Here is a quote from our new Rui et al paper about to be submitted. 
"The impact of a telescope’s FoV and the on-sky observing time is different for surveys of transient (one-off or sporadic) events compared to persistent sources (Cordes, 2007). A survey for sporadic events never ends and the number of events or sources (e.g., FRBs) found is proportional to the product of the observing time and the FoV. In contrast, discovering a new persistent source in a given survey area is only possible with increased sensitivity and that only improves as the square root of the observing time or equivalently the square root of the FoV for a given search area."
I suggest the decal review include a broad case for wide FoV transient astronomy but could then note some specific advantages for the radio version.  It is a development that would be led by Australia (governments like that) and the radio case (along with gravity waves) offers all the time as well as all sky.

**Comment by gemmaanderson** [comment]
This also links with Australia's involvement with the CTA #22 as a transient discovery machine and also multi-messenger follow-up of LSST and SKA/ASKAP/MWA transients.
And indeed an Australian Gravitational wave observatory #14

**Comment by gemmaanderson** [comment]
I will also add that Australia is unique in having the world's widefield radio instruments. I think this is a great point to motivate a lot of the infrastructure and science cases presented in the issue tracker. We are perfectly placed for transient discovery and multi-messenger follow-up (LSST,CTA,LVK, etc).

**Comment by RonEkers** [comment]
Yes, I agree with Gemma that emphasising where Australia has a unique advantage is especially valuable so the decadal review will have impact at a high level.  Great science following others nations is still important but more so for scientists than governments and the government will get the same message from every field of science.

Wide FoV comes naturally in some areas – eg Gravitational wave detectors (maybe the widest FoV of any multi-messenger field) but also high energy detectors either in space or using the atmosphere are intrinsically wide FoV.  Focussing telescopes across most of the EM spectrum can reach extremely high sensitivity but at a huge cost in FoV and in the case of radio Australia happens to be in a leading position for high FoV through both MWA and ASKAP developments.  There immediate payoff in discovery space is very notable on an international scale.  SKA will provide extremely high sensitivity, but not an impressive FoV.
